experiments/react-native-uifabric/src/RNTester/FabricTester.tsx

```diff
@@ -19,70 +14,66 @@ msgq.spy(true);
 
 registerThemes();
 
-const styles = ReactNative.StyleSheet.create({
-  viewStyle: {
-    minHeight: 200,
-    justifyContent: 'space-between'
-  },
-  stackStyle: {
-    borderWidth: 2,
-    borderColor: '#bdbdbd',
-    padding: 12,
-    margin: 8
-  },
-  separatorStackStyle: {
-    height: 200,
-    flexDirection: 'row',
-    justifyContent: 'space-evenly'
+const EmptyComponent: React.FunctionComponent = () => {
+  return <Text fontSize={14} style={fabricTesterStyles.noTest}>Select a component from the left.</Text>;
+}
+
+// sort tests alphabetically by name 
+const sortedTestComponents = allTestComponents.sort(
+  (a, b) => a.name.localeCompare(b.name)
+);
+
+const TestListSeparator = Separator.customize(
+  {
+    tokens: {
+      color: 'darkGray',

```

Comment:
for fun, you could try using one of the themed colors

Reply:
Good idea. I'll do that.